.github/workflows/release-tag-rc.yml

```diff
@@ -0,0 +1,125 @@
+name: release-rc-version
+
+on:
+  push:
+    tags:
+      - 'v1*-rc*'
```

Comment:
what about v2+ tags?

Reply:
I figured I'd keep it specific to v1, and if we ever release a v2 then the workflows may be different.

Reply:
We should start creating major releases instead of minor ones anyways, because those are correct in terms of semver, where each major version contains breaking changes.

Reply:
See https://github.com/go-gitea/gitea/issues/27574 for proposal.